deploy_ocp3_cluster.yml

```diff
@@ -13,11 +13,23 @@
   vars_files:
     - "{{ playbook_dir }}/config/defaults.yml"
 
-- name: Run openshift-ansible prerequisites
+- name: Run the openshift-ansible prerequisites for OCP-3.9-3.11
   import_playbook: "{{ lookup('env', 'WORKSPACE') or '../'}}/openshift-ansible/playbooks/prerequisites.yml"
+  vars:
+    ocp: "{{ lookup('env', 'OPENSHIFT_VERSION') }}"
+  when: ocp == "v3.11"
 
-- name: Run the openshift-ansible depoyment
+- name: Run the openshift-ansible deployment for OCP-3.9-3.11
   import_playbook: "{{ lookup('env', 'WORKSPACE') or '../'}}/openshift-ansible/playbooks/deploy_cluster.yml"
+  vars:
+    ocp: "{{ lookup('env', 'OPENSHIFT_VERSION') }}"
```

Comment:
There is a possibility, that this variable won't be in the current environment at the moment of task execution, so I suggest to consider using something like:
`ocp: "{{ lookup('env', 'OPENSHIFT_VERSION') or 'v3.11'}}"`
This way it is handled in https://github.com/fusor/mig-ci/blob/bb9a1ae50a1ca78ba68e2dc65adf4fa25adda78f/roles/ocp3_aws_deploy/defaults/main.yml#L1

Reply:
done